network/definition/data/templates/signals.yaml

```diff
@@ -95,3 +95,21 @@ signals:
         min: 0.0
         max: 15
     continuous: true
+  tireLinearVelocity:
```

Comment:
the c code all calls it wheel speed but here you're effectively calling it tire speed, maybe pick one and stick to it?